scripts/local-universe.py

```diff
@@ -45,9 +45,9 @@ def main():
     parser.add_argument(
         '--include',
         default='',
-        help='Command separated list of packages to include. If this option '
-        'is not specified then all packages are downloaded. E.g. '
-        '--include="marathon,chronos"')
+        help='Command (,) separated list of packages to include. For each '
```

Comment:
Ah, I think I answered my own question in another comment below. There would be too many packages to include if "not specified" meant "all versions of all packages".